packages/ingest-mongodb-public/src/sources/mongodb-university/makeUniversityPages.ts

```diff
@@ -45,6 +45,32 @@ function makeCatalogItemPages({
 }): Page[] {
   const pages: Page[] = [];
   for (const catalogItem of tiCatalogItems) {
+    /* Create page for higher level courses.
+     * Higher level courses are Leanring Paths and Courses that have nested content.
+     * Nested content is made up of other TiCatalogItems such as Units and Learning Bytes.
+     * Note: Higher level courses do not have videos, but their nested content does.
+     */
+    if (
+      catalogItem.learning_format === "Learning Path" ||
+      catalogItem.learning_format === "Course"
+    ) {
+      const page: Page = {
+        sourceName,
+        url: `https://learn.mongodb.com/learning-paths/${catalogItem.slug}`,
```

Comment:
can you make a `const universityContentBaseUrl = 'https://learn.mongodb.com/'` and use this wherever we use this string? there's 2 uses of it in this PR and i know there's a few more places that we use this in the repo. 

will make future updates easier in case there's ever a base URL change

Reply:
I named it UNI_BASE_URL